### ISSUE TITLE
Blog post for rabies/lassa/yellow-fever

### ISSUE THREAD

**Issue by genehack** [nextstrain/nextstrain.org#1051]
[Draft post](https://docs.google.com/document/d/1UIoLvL8SQ7sDKoPI7Kb9Pcv6K7O8b7SGfAv0tGmwVAo/edit?usp=sharing)

**Comment by genehack** [comment]
@j23414 could you link your in-progress draft here?

**Comment by j23414** [comment]
Yes, can do! In-progress draft is at this [google doc/nextstrain/blogpost/draft](https://docs.google.com/document/d/1UIoLvL8SQ7sDKoPI7Kb9Pcv6K7O8b7SGfAv0tGmwVAo/edit?usp=sharing)
